src/test/java/com/final_assignment/UserMapperTest.java

```diff
@@ -79,14 +78,39 @@ class UserMapperTest {
     }
 
     @Test
-    void insert() {
+    @Sql(
+        scripts = {"classpath:/sqlannotation/delete-users.sql", "classpath:/sqlannotation/insert-users.sql"},
+        executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD
+    )
+    @Transactional
+    void 新規登録ができること() {
+        User newUser = new User("soya", "soya@example.com");
+        userMapper.insert(newUser);
+        List<User> users = userMapper.findAll();
```

Comment:
newUser.getId で採番された ID を取得して、userMaper.findById で ID 検索をかけて取得できること、soyaであることを確認してもいいかなと思いました。
```suggestion
        User newUser = new User("soya", "soya@example.com");
        userMapper.insert(newUser);
        int id = newUser.getId();
        List<User> users = userMapper.findById(id);
```